models/iaf_psc_exp.nestml

```diff
@@ -85,7 +85,7 @@ neuron iaf_psc_exp_neuron:
     shape I_shape_in = exp(-1/tau_syn_in*t)
     shape I_shape_ex = exp(-1/tau_syn_ex*t)
     function  I_syn mV = (curr_sum(I_shape_in, in_spikes) + curr_sum(I_shape_ex, ex_spikes) + I_e + currents)
-    V_abs' = -1/tau_m * V_abs + 1/C_m * I_syn
+    V_abs' = -V_abs/tau_m + (I_syn/C_m) * 1 [s**3 * A**2 /(kg * m**2)] * 1 [s/Gs]
```

Comment:
The units here seem very confusing. I would expect `I_syn` and `C_m` to have their units defined elsewhere, so that the second term can be written without explicit units just as the first. And if units are needed, they should be given as high-level units with suitable prefactors (e.g. mV, pF) instead of basic MSKA and a `G` prefactor.